app/app.go

```diff
@@ -581,9 +581,6 @@ func New(
 
 	app.sm.RegisterStoreDecoders()
 
-	// register upgrade
-	app.RegisterUpgradeHandlers(cfg)
```

Comment:
can you explain this ?

Reply:
I add it after. Only for better readable struct.